spec/features/user_item_view_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+describe "the user item view", type: :feature do
+  before :each do
+    @item = FactoryGirl.create(:item)
+    user = FactoryGirl.create(:user)
+    visit login_path
+    fill_in("session_email", with: user.email)
+    fill_in("session_password", with: user.password)
+    click_button 'Sign in'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.